src/main/java/seedu/address/model/event/DateTime.java

```diff
@@ -180,12 +180,30 @@ public static Optional<LocalTime> parseTime(String dateString) {
         return Optional.ofNullable(generateLocalTime(hours, minutes, seconds));
     }
 
-    public LocalDate getDate() {
-        return this.date;
+    /**
+     * Returns true if a start DateTime is before or equal another DateTime.
+     * Otherwise, returns false.
+     */
+    public boolean isBeforeOrEqual(DateTime other) {
```

Comment:
My take is that actually using the `isBefore()` and `isAfter()` is more explicit, because you don't have to go and think about whether you want a `1` or `0` to equate to. I feel that there's no need to make `DateTime` implement `Comparable`, since the core components of `DateTime`, which are `LocalDate` and `LocalTime` are already implements `Comparable`, and you can in fact just use the methods the Java API provides to do so. 

That said, both implementations will work.